### ISSUE TITLE
Debugging "No buried groups for ~/my/dir

### ISSUE THREAD

**Issue by ddoherty03** [karthink/popper#19]
I am trying to understand why pooper does not associate a buffer with the directory from which it was initiated.  This seems to happen quite often (but not always) for me.  For example, I am now editing in a ruby buffer and I bring up a *devdocs* buffer from the ruby buffer.  I pops up as its supposed to, but if I dismiss it and try to re-display it, I get the message that there are "No buried groups for" that directory.  But if I call up the mode help buffer, it pops up and does get associated with the directory.

Is there a good way to go about diagnosing what is happening here?  I would appreciate some hints.  Here is my config so far:

[popper_config.txt](https://github.com/karthink/popper/files/7793747/popper_config.txt)

**Comment by karthink** [comment]
@ddoherty03 Your config looks fine. Sounds like a bug. After you dismiss the devdocs buffer (and before doing anything else), can you call `eval-expression` and run the following?

```elisp
(alist-get (expand-file-name default-directory) popper-buried-popup-alist)
```

**Comment by ddoherty03** [comment]
@karthink Thanks for the hint.  After I posted, I re-read your README and realized I didn't have the latest version, so I upgraded. 
It now seems to be working great, and I'm excited to get all that side-information organized.  I did your experiment, though after the upgrade, and it yielded nil, but this was a git-controlled buffer, and when I just listed `popper-buried-popup-alist` it showed the devdocs buffer under the project root, which I think is correct.

I am going to close this now, but I may pester you later if I'm puzzled again.  Many thanks for this fantastic package!
